amf-client/shared/src/test/scala/amf/validation/ValidationProfilesCycle.scala

```diff
@@ -50,31 +50,31 @@ class ValidationProfilesCycle extends AsyncFunSuite with PlatformSecrets {
     val exampleFile              = "validation_profile_prefixes.yaml.jsonld"
     val expected: Future[String] = platform.resolve(basePath + expectedFile).map(_.stream.toString)
     val validation               = Validation(platform)
-    cycle(exampleFile, AmfJsonHint, Yaml, Raml).zip(expected).map(checkDiff)
+    cycle(exampleFile, AmfJsonHint, Yaml, Aml).zip(expected).map(checkDiff)
   }
 
   test("Loading and serializing validations with inplace definition of encodes") {
     val expectedFile             = "validation_profile_example_gold.yaml"
     val exampleFile              = "validation_profile_example.yaml"
     val expected: Future[String] = platform.resolve(basePath + expectedFile).map(_.stream.toString)
     val validation               = Validation(platform)
-    cycle(exampleFile, VocabularyYamlHint, Yaml, Raml).zip(expected).map(checkDiff)
+    cycle(exampleFile, VocabularyYamlHint, Yaml, Aml).zip(expected).map(checkDiff)
   }
 
   test("Loading and serializing validations with inplace definition of range") {
     val expectedFile             = "validation_profile_example_gold.yaml"
     val exampleFile              = "validation_profile_example.yaml"
     val validation               = Validation(platform)
     val expected: Future[String] = platform.resolve(basePath + expectedFile).map(_.stream.toString)
-    cycle(exampleFile, VocabularyYamlHint, Yaml, Raml).zip(expected).map(checkDiff)
+    cycle(exampleFile, VocabularyYamlHint, Yaml, Aml).zip(expected).map(checkDiff)
   }
 
   test("Loading and serializing validations with union type") {
     val expectedFile             = "validation_profile_example_gold.yaml"
     val exampleFile              = "validation_profile_example.yaml"
     val validation               = Validation(platform)
     val expected: Future[String] = platform.resolve(basePath + expectedFile).map(_.stream.toString)
-    cycle(exampleFile, VocabularyYamlHint, Yaml, Raml).zip(expected).map(checkDiff)
+    cycle(exampleFile, VocabularyYamlHint, Yaml, Aml).zip(expected).map(checkDiff)
```

Comment:
Weird change

Reply:
This change was made to fix the test.
This is due a change in how the serializer searches for a plugin for rendering. Previously AMFSerializer would search for a plugin that complies with the vendor, mediaType, and unparse method, and as a fallback it searchs for a plugin that complies with the mediaType and unparse (without taking into account the vendor).
Since this logic is now present in the applies method of the render plugin (currently in AMFRenderPluginAdapter), there is no fallback when the vendor is incorrect.